gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AdbTask.groovy

```diff
@@ -70,7 +110,7 @@ public class AdbTask extends org.gradle.api.DefaultTask {
             it.startsWith("$propertyKey:")
         }
         if (output == null) {
-            throw new IllegalStateException("Could not read property '$propertyKey' of $apkPath")
+            throw new IllegalArgumentException("Could not read property '$propertyKey' of $apkPath")
```

Comment:
I had been thinking about that, too, but was a bit undecided. Also line 23 (`getPackageName()`) would require fixing. I'll push a version that returns `null` anyway, and then you can choose :-)